.github/workflows/go.fmt.yml

```diff
@@ -45,6 +45,13 @@ jobs:
     steps:
     - name: Checkout
       uses: actions/checkout@master
+    - name: Check Go Version
+      run: echo "::set-output name=value::$(cat GO_VERSION)"
+      id: go-version
+    - name: Install Go
+      uses: actions/setup-go@v2
+      with:
+        go-version: ${{ steps.go-version.outputs.value }}
```

Comment:
No need to set up Go on the GitHub Runner, since the *go-fmt* action runs within Docker.

Reply:
Removed Go setup.